src/index.html

```diff
@@ -1,13 +1,65 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+    <div class="product" data-qa="card">
+      <div class="product__image-block">
+        <img
+        src="images/imac.jpeg"
+        alt="monitor-picture"
+        class="product__image"
+      >
+      </div>
+      <div class="product__titles">
+        <span class="product__item-title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </span>
+        <span class="product__item-code">
+          Код товара: 195434
+        </span>
+      </div>
+
+      <div class="product__feedback">
+        <div class="product__stars">
+          <div class="product__star-active"></div>
+          <div class="product__star-active"></div>
+          <div class="product__star-active"></div>
+          <div class="product__star-active"></div>
+          <div class="product__star-not-active"></div>
```

Comment:
```suggestion
          <div class="product__star--active"></div>
          <div class="product__star--not-active"></div>
```
'active' and 'not-active' are modifiers